libs/api/domains/cms/src/lib/search/contentful.service.ts

```diff
@@ -183,14 +196,39 @@ export class ContentfulService {
       deletedEntries,
     } = await this.getSyncData(typeOfSync)
 
+    const processEntries = entries
+      .filter((entry) => entry.sys.contentType.sys.id === 'processEntry')
+      .map((entry) => entry.sys.id)
+
```

Comment:
This needs to cast a wider net `entry.sys.contentType.sys.id === 'processEntry'` -> `nestedContentTypes.includes(entry.sys.contentType.sys.id)`
Where `nestedContentTypes` is a list of all content types nested in all page types